doc/source/astrometry.rst

```diff
@@ -1 +1,246 @@
 .. _astrometry:
+
+===================================================
+Headerlets and You: Astrometry in Drizzle Products
+===================================================
+
+The astrometry for any given observation relies upon accurate pointing information from the telescope.   However, HST has evolved over time since it was put into orbit, with the focus changing over time as the telescope desorbs.  The instruments have also changed positions over time relative to the FGS guides, and the coordinates for the guide stars were orginally determined using ground-based information.  All this has limited the calculation of the pointing of any given observation on the sky (absolute astrometry) to no better than 1-2 arc-seconds.
+
+Calibration of the distortion model for each instrument has been done well enough to allow observations to be combined (relative astrometry) with an accuracy of better than 5 milli-arcseconds.  This has made the absolute astrometry inaccuracy stand out even more, making it more difficult to compare observations taken at different times or to compare HST data with observations from other telescopes (like Chandra).
+
+Therefore, multiple efforts have been undertaken to improve the absolute astrometry to match the accuracy of the relative astrometry.  These efforts have resulted in multiple new world coordinate systems (WCS) solutions to be developed for HST observations, starting with ACS, WFC3, and WFPC2.
+
+Each solution has its own advantages and errors, making some good for one use but inadequate for others.  As a result,  **all new WCS solutions which are approved by STScI** are being offered with HST data provided by MAST with headerlets serving as the mechanism for providing and applying all WCS solutions.
+
```

Comment:
It might be beneficial to include 1 or more citations to the ISRs, or specific sections of data/instrument handbooks that go into the evolving accuracy of absolute astrometry in further detail.

Reply:
I will add one reference for each instrument which summarizes the latest calibrations of the distortion models and its use in drizzling the images.  These references also include pointers to the latest ISRs.